src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -1219,4 +1168,202 @@ public void stop() {
          keepRunning = false;
      }
 
+     /**
+      * Invokes the PC2 server to update the status of the specified run to match the specified judgement.
+      *
+      * @param submissionIdStr the Id of the run to be updated.
+      * @param newJudgementStr the judgement which should be applied to the specified run.
+      *
+      * @return true if the run was successfully updated; false if the run could not be updated for some reason
+      */
+     protected boolean updateRun(String submissionIdStr, String newJudgementStr) {
```

Comment:
It does not appear this method, added in the most recent commit, is referenced anywhere. Am I missing something, or should it be removed?